profiles/cli.py

```diff
@@ -36,6 +38,44 @@ def run(self) -> None:
             each.clear()
 
 
+class CreateProfileCommand(Command):
+    """Allow manual creation of profiles from the commandline.
+
+    Example:
+
+    $ python manage.py create-profile --name "Test User" --email "test@test.com"
+
+    """
+    NAME = 'create-profile'
+
+    def __init__(self, profiles: Profiles):
+        super(CreateProfileCommand, self).__init__()
+        self.profiles = profiles
+
+    def get_options(self) -> List[Option]:
+        return [
+            Option('-e', '--email', dest='email'),
+            Option('-n', '--name', dest='name'),
+        ]
+
+    # pylint: disable=method-hidden,arguments-differ
+    def run(self, name, email) -> None:
+        # pylint is complaining about arguments differing from base class
+        # though this is the recommended way to define the option args in the
+        # run() definition...
+        if name and email:
+            try:
+                profile = self.profiles.get_by_email_address(email)
+            except ProfileNotFound:
+                profile = Profile(self.profiles.next_id(), Name(name.strip('"')))
+                self.profiles.add(profile)
+                profile.add_email_address(email=email, restricted=True)
+
+            return profile.id, email
```

Comment:
when is the transaction committed? Would be ideal to do so before returning the id to the user (otherwise a transaction failure would make that `id` vanish)

Reply:
We are using `SQLALCHEMY_COMMIT_ON_TEARDOWN = True` by default in the config. We, therefore, do not manually commit the session anywhere in the app. As the command supports re running with the same data (perhaps due to an error?) is this OK for now?